kubectl-cloudflow/cloudflowavro/avrocompatibility_test.go

```diff
@@ -0,0 +1,109 @@
+package cloudflowavro
+
+import (
+	"github.com/hamba/avro"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"testing"
+)
+
+func Test_SimpleSchemaCompatibility(t *testing.T) {
```

Comment:
Did we decided on using Ginkgo for all tests or just in specific situations?

Reply:
I propose to use it for all BDD like tests that express high level cloudflow logic eg. blueprint spec.

Reply:
ok, and then we use standard tests for the rest, check